community-plugins.json

```diff
@@ -5707,7 +5707,7 @@
     {
         "id": "contextual-comments",
         "name": "Contextual Comments",
-        "description": "General comments or langage related (in code blocks) + 2 commands trim end lines (all doc/code blocks).",
+        "description": "General comments or langage related (js, python, C++, dataviewjs, lua...) Fast and 100% efficient.",
```

Comment:
Typo in the word language, also make sure that the capitalization of the language names is correct